recipe/meta.yaml

```
@@ -1,22 +1,22 @@
-{% set version = "2.7.0" %}
+{% set version = "2.8.0" %}
+{% set io_gcs_version = "0.24.0" %}
 
 package:
   name: tensorflow-split
   version: {{ version }}
 
 source:
   - url: https://github.com/tensorflow/tensorflow/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: bb124905c7fdacd81e7c842b287c169bbf377d29c74c9dacc04f96c9793747bb
+    sha256: 66b953ae7fba61fd78969a2e24e350b26ec116cf2e6a7eb93d02c63939c6f9f7
     patches:
-      - patches/0001-relax-tolerence-in-special_math_test.patch
-      - patches/0002-skip-read-only-debugger_cli_common-test.patch
-      - patches/0003-relax-precision-in-matrix_log_op_test.patch
-      # requires setting GRPCIO_VERSION in build script
-      - patches/0004-loosen-requirements.patch
-      - patches/0005-Replace-deprecated-absl-Status-error_message-call-wi.patch
+      - patches/0001-loosen-requirements.patch
+
   - url: https://github.com/tensorflow/estimator/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: e5164e802638d3cf110ecc17912be9d514a9d3354ec48e77200b9403dcc15965
+    sha256: 58a2c3562ca6491c257e9a4d9bd8825667883257edcdb452181efa691c586b17
     folder: tensorflow-estimator
+  - url: https://github.com/tensorflow/io/archive/refs/tags/v{{ io_gcs_version }}.tar.gz
+    sha256: 7a4b57f6f438402bab4919c360e931c32f4d8d8afa23116d32a48461dddd91dc
+    folder: tensorflow-io
 
 build:
   number: 0
```

```suggestion
  number: 1
```

increase the build number as this would be the second release when it is fixed and approved.